test/tools/ossfuzz/strictasm_diff_ossfuzz.cpp

```diff
@@ -97,6 +101,13 @@ extern "C" int LLVMFuzzerTestOneInput(uint8_t const* _data, size_t _size)
 		(yul::test::yul_fuzzer::yulFuzzerUtil::maxSteps * 4)
 	);
 
+	if (
+		termReason == yulFuzzerUtil::TerminationReason::StepLimitReached ||
+		termReason == yulFuzzerUtil::TerminationReason::TraceLimitReached ||
+		termReason == yulFuzzerUtil::TerminationReason::ExpresionNestingLimitReached
```

Comment:
Not necessarily, but maybe worth extracting this full condition into a function? In case it's reused more and more. Also fine as is, so will approve and leave the decision to you.

Reply:
Thanks, updated.